elements/pl-file-upload/pl-file-upload.mustache

```diff
@@ -1,15 +1,13 @@
 <script>
-$(function() {
-  var UUID = "{{uuid}}";
-
-  new window.PLFileUpload(UUID, {
-    acceptedFiles: {{&file_names}}
-    {{#has_files}}, files: {{&files}}{{/has_files}}
+$(() => {
+  new window.PLFileUpload("{{uuid}}", {
+    acceptedFiles: {{&file_names}},
+    {{#has_submitted_files}}submittedFileNames: {{&submitted_file_names}},{{/has_submitted_files}}
```

Comment:
Is the `has_submitted_files` key necessary? Can't you just case on whether `submitted_file_names` is empty?

Reply:
I sure can! This was based on existing code.